mypyc/irbuild/function.py

```diff
@@ -246,7 +246,15 @@ def c() -> None:
         # Do a first-pass and generate a function that just returns a generator object.
         gen_generator_func(builder)
         args, _, blocks, ret_type, fn_info = builder.leave()
-        func_ir, func_reg = gen_func_ir(builder, args, blocks, sig, fn_info, cdef)
+        func_ir, func_reg = gen_func_ir(
+            builder,
+            args,
+            blocks,
+            sig,
+            fn_info,
+            cdef,
+            is_singledispatch,
+        )
```

Comment:
This can probably fit onto one line, right?

Reply:
I split it into multiple lines because flake8 was saying that line was too long. I guess I can try fitting it in 2 lines if you want me to.

Reply:
I would probably go with
```
        func_ir, func_reg = gen_func_ir(
            builder, args, blocks, sig, fn_info, cdef, is_singledispatch
        )
```

mypy is fairly inconsistent about how we break lines, though, so you could probably get away with something else if you like it more :P

Reply:
That does look better. I'll change that.